packages/http-server/src/__tests__/integration/http-server.integration.ts

```diff
@@ -264,6 +302,17 @@ describe('HttpServer (integration)', () => {
     res.end();
   }
 
+  function createRequestHandler(delay: number) {
+    return function dummyRequestHandlerWithDelay(
+      req: IncomingMessage,
+      res: ServerResponse,
+    ): void {
+      setTimeout(() => {
+        res.end();
+      }, delay);
```

Comment:
Hard-coded timeouts are brittle and can lead to intermittent CI failures. Can you find a more robust solution please? (See https://github.com/strongloop/loopback-next/pull/4146#discussion_r347771308).

I like the solution you found for waiting until a request is accepted.

Let's please find a solution that will allow the request handler to avoid timeouts too.